index.html

```diff
@@ -2031,6 +2051,62 @@ <h2>
           Visual representation of a [=Standard Gamepad=] layout.
         </figcaption>
       </figure>
+      <section data-dfn-for="ExtendedMapping" data-link-for="ExtendedMapping">
+        <h3>
+	  <dfn>ExtendedMapping</dfn>
+        </h3>
+        <p>
+          This section introduces an extended gamepad button mapping beyond the Standard Gamepad layout.
+          These additional buttons are commonly found on certain gamepad models.
+          The following table defines the extended buttons used by some gamepad models.
+          It's important to note that this list is not exhaustive, and user agents may utilize different
+          or additional buttons for these or other gamepad models.
+          Consequently, the number of buttons on the {{Gamepad}} is not limited to the standard mapping of 17 buttons.
+        </p>
+       <p>
+          To accommodate extended gamepads, we have incorporated an "extended" mapping enumeration
```

Comment:
It seems the "extended" value hasn't been added to GamepadMappingType yet. Can you please update the PR and add it,  along with this definition?